hack/python-release.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+cd ./python
+
+packages=$(find . -maxdepth 1 -type d)
+
+for folder in ${packages[@]}
+do
+    echo "moving into folder ${folder}"
+    if [[ ${folder} == 'plugin' || ! -f "${folder}/pyproject.toml" ]]; then
```

Comment:
note that this script requires installing the plugin with
`pip install ./python/plugin/poetry-version-plugin/`